test/unit_tests/unittest_scanner.cpp

```diff
@@ -39,50 +37,12 @@ static const std::string DEVICE_IP{ "127.0.0.100" };
 static constexpr double START_ANGLE{ 0. };
 static constexpr double END_ANGLE{ degreeToRad(275.) };
 
-static constexpr std::chrono::milliseconds DEFAULT_TIMEOUT{ 50 };
+using MockedScanner = ScannerT<psen_scan_v2_test::ScannerControllerMock>;
 
-class ScannerTest : public testing::Test
+TEST(ScannerTest, testGetCompleteScan)
 {
-protected:
-  ScannerConfiguration scanner_config_{ HOST_IP,   HOST_UDP_PORT_DATA, HOST_UDP_PORT_CONTROL,
-                                        DEVICE_IP, START_ANGLE,        END_ANGLE };
-};
-
-typedef ScannerT<psen_scan_v2_test::ScannerControllerMock> MockedScanner;
-
-TEST_F(ScannerTest, testStart)
```

Comment:
Why did you remove these tests?

Reply:
Because they are a duplication with regard to the tests executed in the ScannerAPI integrationtest. -> Done.